src/components/Selector/Selector.tsx

```diff
@@ -5,16 +5,25 @@ interface SelectorProps {
   title?: string
   type?: string
   min?: number
+  placeholder?: string
 }
 
-function Selector({ listId, title, type, min = 1 }: SelectorProps) {
+function Selector({
+  listId,
+  title,
+  type,
+  min = 1,
+  placeholder,
+}: SelectorProps) {
   return (
     <S.Container>
-      <S.Title>{title}</S.Title>
+      <S.Title htmlFor={title}>{title}</S.Title>
```

Comment:
Suponho que `htmlFor` seja para acessibilidade, correto?

Reply:
sim, a depender de como a tag label seja colocada, ela pode precisar de dois atributos, 'id' do input que ela está sendo associada e o 'for(no react: htmlFor)', para passar esse 'id'